test/functional/rawtransactions.py

```diff
@@ -63,7 +63,7 @@ def run_test(self):
         addr2Obj = self.nodes[2].validateaddress(addr2)
 
         mSigObj = self.nodes[2].addmultisigaddress(2, [addr1Obj['pubkey'], addr2Obj['pubkey']])
-        mSigObjValid = self.nodes[2].validateaddress(mSigObj)
+        assert self.nodes[2].validateaddress(mSigObj)
```

Comment:
IIRC, this returns a dict and you'd need to assert on the value of the "isvalid" key.